Patch-Source: https://github.com/LudovicRousseau/PCSC/commit/960e367edc01518b90684ffff99010ee07ae1bd4
---
From 960e367edc01518b90684ffff99010ee07ae1bd4 Mon Sep 17 00:00:00 2001
From: Ludovic Rousseau <ludovic.rousseau@free.fr>
Date: Sat, 4 May 2024 15:39:39 +0200
Subject: [PATCH] meson: include prefix in sbindir

If sbindir is set to "bin" and prefix is set to "/usr" then sbindir is
"/usr/bin" instead of "/bin".

Thanks to Frederik Schwan for the bug report
" [Build] Meson ignoring prefix for binary location #179 "
https://github.com/LudovicRousseau/PCSC/issues/179
---
 meson.build | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 843d643e..0686a151 100644
--- a/meson.build
+++ b/meson.build
@@ -20,11 +20,12 @@ conf_data = configuration_data({
   'NO_LOG' : get_option('embedded'),
   })
 # for generated files from .in templates
+sbindir = join_paths(get_option('prefix') , get_option('sbindir'))
 confgen_data = configuration_data({
   'VERSION' : meson.project_version(),
   'ipcdir' : get_option('ipcdir'),
   'sysconfdir' : get_option('sysconfdir'),
-  'sbindir_exp' : '/' / get_option('sbindir'),
+  'sbindir_exp' : sbindir,
   'PCSCLITE_CONFIG_DIR' : get_option('serialconfdir'),
   'usbdropdir' : get_option('usbdropdir'),
   })
@@ -143,7 +144,7 @@ executable('pcscd',
   dependencies : pcscd_dep,
   c_args: '-DPCSCD',
   export_dynamic : true,
-  install_dir : '/' / get_option('sbindir'),
+  install_dir : sbindir,
   install : true)
 
 # libpcsclite_real library
